CMakeLists.txt

```diff
@@ -16,35 +16,43 @@ set(MACOS_BUNDLEID "com.example.${CMAKE_PROJECT_NAME}")
 # Replace `me@contoso.com` with the maintainer email address you want to put in Linux packages
 set(LINUX_MAINTAINER_EMAIL "me@mymailhost.com")
 
+# Enable this option if the plugin uses Qt functionality.
+#
+# * WARNING: This adds a dependency of the plugin to the Qt version shipped with OBS.
+#
+# * NOTE: .ui files still need to be added to target_sources.
+#
+option(ENABLE_UI "Enable UI functionality" OFF)
+
 # Add your custom source files here - header files are optional and only required for visibility
 # e.g. in Xcode or Visual Studio
 target_sources(${CMAKE_PROJECT_NAME} PRIVATE src/plugin-main.c)
 
-# /!\ TAKE NOTE: No need to edit things past this point /!\
-
+# Import libobs as main plugin dependency
 find_package(libobs REQUIRED)
-find_package(obs-frontend-api REQUIRED)
 include(cmake/ObsPluginHelpers.cmake)
-find_qt(COMPONENTS Widgets Core)
+
+# Uncomment those lines if you want to use the OBS Frontend API in your plugin
+# find_package(obs-frontend-api REQUIRED)
+
+# target_link_libraries(${CMAKE_PROJECT_NAME} PRIVATE OBS::obs-frontend-api)
+
+# Uncomment those lines if you want to use Qt in your plugin find_qt(COMPONENTS Widgets Core)
```

Comment:
cmake-format need more newline to not do that.

```
# Uncomment those lines if you want to use Qt in your plugin

# find_qt(COMPONENTS Widgets  Core)
```

Reply:
Used multiline comments to avoid CMake-Format interfering with it too much. Pls check.

Reply:
From what I saw multi-line comment ends only with `]]` not `#]]`.

Reply:
Yes, because the hash before is effectively deemed part of the multi-line comment. It is not necessary, but it makes more sense for users to understand that they have to remove that line to uncomment.